src/main/resources/biz_en.properties

```diff
@@ -508,6 +508,7 @@ PersonData.title = Title
 Process = Process
 Process.completed = Completed
 Process.counters = Performance
+Process.errorneous = Errorneous
```

Comment:
```suggestion
Process.errorneous = Erroneous
```

Reply:
Huh, seems like the field is named wrongly